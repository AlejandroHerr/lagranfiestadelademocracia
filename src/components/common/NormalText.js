import React from 'react';
import PropTypes from 'prop-types';
import { Text } from 'rebass';

const NormalText = ({ children, ...props }) => (
  <Text fontFamily="body" {...props}>
    {children}
  </Text>
);

NormalText.propTypes = {
  children: PropTypes.string.isRequired,
};

export default NormalText;
