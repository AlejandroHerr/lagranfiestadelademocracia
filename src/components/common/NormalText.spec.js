import React from 'react';
import renderer from 'react-test-renderer';
import NormalText from './NormalText';

describe('components/component/NormalText', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NormalText>Test Text</NormalText>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
