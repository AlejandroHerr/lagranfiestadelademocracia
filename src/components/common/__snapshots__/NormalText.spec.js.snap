// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/component/NormalText renders correctly 1`] = `
<div
  className="sc-bdVaJa sc-htpNat jBGZHO"
  fontFamily="body"
>
  Test Text
</div>
`;
